bigquery_etl/braze_currents/__init__.py

```diff
@@ -31,7 +31,7 @@ def import_braze_current_from_bucket(
 ):
     """Use bigquery client to store AVRO files from bucket in BigQuery."""
     client = bigquery.Client(destination_project)
-    uri = f"gs://{source_bucket}/{source_prefix}/event_type={event_type}"
+    uri = f"gs://{source_bucket}/{source_prefix}/event_type={event_type}/*"
```

Comment:
A couple of belated thoughts:
- If the files have a consistent file extension (e.g. `.avro`) then specifying that in the URI pattern could make sense as a safeguard in case Braze ever puts other kinds of files there (e.g. readmes, metadata files).
- This is reloading all data from the Braze files every time.  If there's a straightforward way to make this incremental that could be beneficial in terms of performance and data integrity (in case Braze removes historical files).  Or another approach to that would be having downstream ETLs persisting the data incrementally.